libraries/networking/src/webrtc/WebRTCSignalingServer.cpp

```diff
@@ -15,61 +15,83 @@
 #include <QWebSocket>
 
 #include <BuildInfo.h>
-#include <PathUtils.h>
 
 #include "../NetworkLogging.h"
 #include "../NodeType.h"
 
 
 const int WEBRTC_SOCKET_CHECK_INTERVAL_IN_MS = 30000;
 
-WebRTCSignalingServer::WebRTCSignalingServer(QObject* parent, bool isWSSEnabled) :
+std::pair<bool, QSslConfiguration> getSSLConfiguration(const CertificatePaths& certPaths) {
+    bool useSystemDefaultCA = certPaths.trustedAuthorities != "";
+
+    qCDebug(networking_webrtc) << "WebSocket WSS key file:" << certPaths.key;
+    qCDebug(networking_webrtc) << "WebSocket WSS cert file:" << certPaths.cert;
+    if(useSystemDefaultCA) {
```

Comment:
```suggestion
    if (useSystemDefaultCA) {
```

Reply:
The useSystemDefaultCA seems to have been all wrong there o_o fixed that along with the formatting issues.